### ISSUE TITLE
fails to attach screenshot on failure, when running on android or iphone

### ISSUE THREAD

**Issue by dmerc** [mucsi96/nightwatch-cucumber#234]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead use https://groups.google.com/d/forum/nightwatch-cucumber
```

**Current behavior**

when running nightwatch on browerstack, using android or iphone device,
and nightwatch config has screenshots enabled on error and on failure, like example:

```
		test_settings: {
			'default': {
				screenshots: {
					enabled: true,
					on_failure : true,
					on_error : true,
					path: 'somepath'
				},
			},
		}
```

```nightwatch-cucumber/lib/nightwatch-api.js``` in the ```takeScreenshot``` function throws this error
```new Error('Creating screenshot was not successful. Response was:\n' + require('util').inspect(response))```

then dumps whole screenshot base64 to console. This happens because somewhy on android & ios
```response.state !== 'success'``` but rather it has ```response.status == 0```.
So i suggest to modify the if condition to:
```if (response.state !== 'success' && response.status !== 0) {```

**Comment by mucsi96** [comment]
Thanks for your feedback! Can you send a pull request for this?

**Comment by dmerc** [comment]
sure, please see #241

**Comment by mucsi96** [comment]
Released in 7.1.10. Thanks!

**Comment by dmerc** [comment]
cheers !
